test/acceptance/authz/backups_test.go

```diff
@@ -112,24 +96,15 @@ func TestAuthZBackupsManageJourney(t *testing.T) {
 		require.Contains(t, parsed.Payload.Error[0].Message, "forbidden")
 	})
 
-	t.Run("fail to get create backup status due to missing read_backups action", func(t *testing.T) {
-		_, err := helper.CreateBackupStatusWithAuthz(t, backend, backupID, "", "", helper.CreateAuth(customKey))
-		require.NotNil(t, err)
-		parsed, forbidden := err.(*backups.BackupsCreateStatusForbidden)
-		require.True(t, forbidden)
-		require.Contains(t, parsed.Payload.Error[0].Message, "forbidden")
-	})
-
-	t.Run("fail to get restore backup status due to missing read_backups action", func(t *testing.T) {
+	t.Run("fail to get restore backup because it wasn't created in 1st place", func(t *testing.T) {
 		_, err := helper.RestoreBackupStatusWithAuthz(t, backend, backupID, "", "", helper.CreateAuth(customKey))
 		require.NotNil(t, err)
-		parsed, forbidden := err.(*backups.BackupsRestoreStatusForbidden)
-		require.True(t, forbidden)
-		require.Contains(t, parsed.Payload.Error[0].Message, "forbidden")
+		_, notfound := err.(*backups.BackupsRestoreStatusNotFound)
+		require.True(t, notfound)
 	})
```

Comment:
I think we can remove this test within the journey since it doesn't test any authz, wdyt?